learn.html

```diff
@@ -0,0 +1,447 @@
+							<p class="c_snip">remote@&#60;host&#62;:&#60;label&#62;</p>
+							<p class="info_text" style="text-align: justify">The label is optionally specified with <b>--label</b> and is empty by default.</p>
+							<p class="info_text" style="text-align: justify">When pbench-start-tools</b> is called on the controller, it starts the local collection (if any), but it also interprets the above stanzas and starts the appropriate tools on the remote hosts. Similarly for pbench-stop-tools and <span class="c_snip_inline">pbench-postprocess-tools</span>.</p>
+						</div>
+					</div>
+				</div>
+				<div class="accordionCard">
+					<h3 class="accordion">
+						<button class="accordionBtn pf-c-accordion__toggle" aria-expanded="true">
+							<i class="arrowIcon fas fa-angle-right pf-c-accordion__toggle-icon" aria-hidden="true"></i>
+							<span class="pf-c-accordion__toggle-text">What does --label do?</span>
+						</button>
+					</h3>
+					<div class="explanation pf-c-accordion__expanded-content pf-m-fixed" hidden>
+						<div class="pf-c-accordion__expanded-content-body">
+							<p class="info_text" style="text-align: justify">TBD</p>
+						</div>
+					</div>
+				</div>
+				<div class="accordionCard">
+					<h3 class="accordion">
+						<button class="accordionBtn pf-c-accordion__toggle" aria-expanded="true">
+							<i class="arrowIcon fas fa-angle-right pf-c-accordion__toggle-icon" aria-hidden="true"></i>
+							<span class="pf-c-accordion__toggle-text">How to add a collection tool</span>
+						</button>
+					</h3>
+					<div class="explanation pf-c-accordion__expanded-content pf-m-fixed" hidden>
+						<div class="pf-c-accordion__expanded-content-body">
+							<p class="info_text" style="text-align: justify">Tool scripts are mostly boilerplate: they need to take a standard set of commands (–install, –start, –stop, –postprocess) and a standard set of options (–iteration, –group, –dir, –interval, –options). Consequently, the easiest thing to do is to take an existing script and modify it slightly to call the tool of your choice. I describe here the case of turbostat.</p>
+							<p class="info_text" style="text-align: justify">There are some tools that timestamp each output stanza; there are others that do not. In the former case, make sure to use whatever option the tool requires to include such timestamps (e.g. vmstat -t on RHEL6 or RHEL7 - but strangely <b>not</b> on Fedora 20 - will produce such timestamps).</p>
+							<p class="info_text" style="text-align: justify">There are some tools that are included in the default installation - others need to be installed separately. Turbostat is not always installed by default, so the tool script installs the package (which is named differently on RHEL6 and RHEL7) if necessary. In some cases (e.g. the sysstat tools), we provide an RPM in the pbench repo and the tool script makes sure to install that if necessary.</p>
```

Comment:
I don't believe that `turbostat` is installed by the tool script; we check whether it has been installed externally, though.

Reply:
We need to address this comment in a separate cleanup of the documentation for the agent in the upcoming release.  We can address this then.